### DISCUSSION TITLE
Copilot For Student

### DISCUSSION THREAD

**Discussion by jmichalenko** [community/community#31376] [GitHub Education]
Hello, Does anyone know how to get copilot for free for students.  I have a verified education account, but when I try to sign up for copilot, it says that students should be free, but I am still only given the option for 60 day trial/$10 per month after that.  Any suggestions, or alternate links? Thank you.

**Comment by zoranduric** [comment]
Make sure that you are verified student. you can check it at [student verification page](https://education.github.com/discount_requests/student_application)

**Reply by jmichalenko** [reply]
Thank you for the response, yes, I am verified.

**Reply by zoranduric** [reply]
should work, have you installed copilot extension in your editor?

**Comment by jmichalenko** [comment]
Hello, Thank you again for response;

Before I got your message, I actually put in credit card info, and then cancelled my trial.  So everything is working, but I guess I'll have to wait and see if it continues to work beyond the trial. 

Thank you!
